neqo-qpack/src/encoder.rs

```diff
@@ -438,6 +460,36 @@ mod tests {
         }
     }
 
+    fn connect_flow_control(max_data: u64) -> TestEncoder {
```

Comment:
This is a lot of duplicate code.  Maybe you should have a baseline function like:

```rust
fn connect_generic<F>(f: F) -> TestEncoder 
where F : FnOnce(&mut Connection,&mut Connection) {
        let mut conn = default_client();
        let mut peer_conn = default_server(); 
        f(&mut conn, &mut peer_conn);
        
         // ...
}
```
This could then be:
```rust
fn connect_fc(max_data: u64) -> TestEncoder {
  connect_generic(|client, server| {
     server.set_local_tparam(...);
  })
```